package.json

```diff
@@ -72,7 +72,7 @@
     "webpack-cli": "^4.7.0"
   },
   "resolutions": {
-    "**/@types/react": "^18.0.14",
```

Comment:
Do we no longer need to resolve React? This has always been painful in the past...

Reply:
Well, we might need to re-add it back depending on what happens with React 19, but for now I'd prefer to remove it 😁. (Yarn 3's syntax for this is also somewhat different so the `"**/@types/react"` is actually no longer valid).